packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/prepare-app/index.md

```diff
@@ -63,4 +63,14 @@ Click **Test Connector Configuration** to confirm that Okta can connect to your
 1. Click **Save** to complete the SCIM integration setup.
 -->
 
+## Troubleshooting
+
+If you experience any difficulties when creating your SCIM integration in Okta, check out the system log information available in the Okta Admin Console.
+
+1. From the Admin Console for your developer org, go to **Applications > Applications**.
+1. Click on your Okta integration to open the integration settings page.
```

Comment:
Click your Okta integration to open the integration settings page.

or

Select your Okta integration to open the integration settings page.